### DISCUSSION TITLE
NaverMapMarkerOverlay 내 사용자 정의 React View

### DISCUSSION THREAD

**Discussion by jun-jaehyuk-lee** [mj-studio-library/react-native-naver-map#91] [Q&A]
안녕하세요. fabric 도입을 뒤로하고 다시 1.5.9 버전으로 직접 클러스터를 구현중에 있습니다.

5. 사용자 정의 React View(캐싱 ❌)
이 유형은 매우 생성될 수 있는 성능에 영향을 미칠 수 있습니다. 아직은 익명으로 사용하거나, 아니면 이미지를 사용하는 것을 추천합니다.

1 버전에 구현해주신 부분에서 더많은 기능이 필요하게되어 직접 클러스터를 구현중에 있습니다.
클러스터기능을 구현하려면 필연적으로 NaverMapMarkerOverlay 안에 사용자 정의 React View 가 들어가야하는거로 확인했습니다.
허나 위 5번 이슈때문인지 최적화를 해보아도 안드로이드에서 깜빡거림현상이 지속되서 발생됩니다. (ios에서는 15pro기준으로는 괜찮은거같습니다.)

향후 1버전(구 아키텍쳐)에 5. 사용자 정의 React View(캐싱 ❌) 해당부분에 대한 업데이트가 예정되어있으실까요? 

감사합니다.

@mym0404

**Comment by mym0404** [comment]
안녕하세요, 구 버전 아키텍처에 대한 지원은 유지보수적인 측면을 고려했을 때 중단되었습니다. 안드로이드에서 커스텀 마커의 렌더링에서 깜빡임이 발생하는 현상은 새로운 아키텍처에서도 발생하는지 모르겠지만 정확한 원인을 아직 파악하지 못한 상태입니다.

이런 현상이 지속적으로 발생된다면 뷰를 관리하는 것이 아닌 뷰의 이미지를 생성해 보여주는 방식으로 변경될 수도 있을 것 같습니다.

**Reply by jun-jaehyuk-lee** [reply]
답변 감사드립니다.
